code/campaignclassic/src/main/java/com/adobe/marketing/mobile/BasicTemplateNotificationBuilder.java

```diff
@@ -0,0 +1,175 @@
+/*
+  Copyright 2023 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+package com.adobe.marketing.mobile;
+
+import android.app.Notification;
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Color;
+import android.os.Build;
+import android.widget.RemoteViews;
+import androidx.annotation.NonNull;
+import androidx.core.app.NotificationCompat;
+import com.adobe.marketing.mobile.campaignclassic.R;
+import com.adobe.marketing.mobile.services.Log;
+import com.adobe.marketing.mobile.services.ServiceProvider;
+import com.adobe.marketing.mobile.util.StringUtils;
+
+public class BasicTemplateNotificationBuilder {
+    private static final String SELF_TAG = "BasicTemplateNotificationBuilder";
+
+    @NonNull static Notification build(final BasicPushTemplate pushTemplate, final Context context) {
+        Log.trace(
+                CampaignPushConstants.LOG_TAG,
+                SELF_TAG,
+                "Building a basic template push notification.");
+        final String channelId =
+                AEPPushNotificationBuilder.createChannelAndGetChannelID(pushTemplate, context);
+        final String packageName =
+                ServiceProvider.getInstance()
+                        .getAppContextService()
+                        .getApplication()
+                        .getPackageName();
+        final RemoteViews smallLayout =
+                new RemoteViews(packageName, R.layout.push_template_collapsed);
+        final RemoteViews expandedLayout =
+                new RemoteViews(packageName, R.layout.push_template_expanded);
+
+        // get push payload data
+        final String imageUrl = pushTemplate.getImageUrl();
+        if (!StringUtils.isNullOrEmpty(imageUrl)) {
+            final Bitmap image = CampaignPushUtils.download(imageUrl);
+            if (image != null) {
+                smallLayout.setImageViewBitmap(R.id.template_image, image);
```

Comment:
The small layout doesn't have to have an image (although it can have one to possibly show a preview of the image to be shown in the expanded layout). If there is no image then you are correct, the only difference is the text.